src/modules/fancyzones/lib/ZoneSet.h

```diff
@@ -1,14 +1,9 @@
 #pragma once
 
 #include "Zone.h"
+#include "JsonHelpers.h"
+#include "util.h"
 
-enum class ZoneSetLayout
-{
-    Grid,
-    Row,
-    Focus,
-    Custom
-};
 
 interface __declspec(uuid("{E4839EB7-669D-49CF-84A9-71A2DFD851A3}")) IZoneSet : public IUnknown
```

Comment:
I wonder why it's a COM interface at all if it's full of STL classes in its methods' arguments.